pytripgui/view/main_window.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'main_window.ui'
+#
+# Created by: PyQt5 UI code generator 5.9.2
```

Comment:
I don't like commiting autogenerated files, for packaging this file can be automatically generated.
Developers can also do it on their own.

Reply:
I thought so too. Where to place generation scrpits?

Reply:
Best - outside pytripgui directory, if small in a main dir, elsewhere in some subdirectory.